src/store_rebuild.cc

```diff
@@ -152,18 +153,18 @@ storeRebuildComplete(StoreRebuildData *dc)
 
     const auto dt = tvSubDsec(counts.startTime, current_time);
 
-    debugs(20, DBG_IMPORTANT, "Finished rebuilding storage from disk.");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.scancount  << " Entries scanned");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.invalid  << " Invalid entries.");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.badflags  << " With invalid flags.");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.objcount  << " Objects loaded.");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.expcount  << " Objects expired.");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.cancelcount  << " Objects cancelled.");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.dupcount  << " Duplicate URLs purged.");
-    debugs(20, DBG_IMPORTANT, "  " << std::setw(7) << counts.clashcount  << " Swapfile clashes avoided.");
-    debugs(20, DBG_IMPORTANT, "  Took "<< std::setw(3)<< std::setprecision(2) << dt << " seconds ("<< std::setw(6) <<
+    debugs(20, Important(46), "Finished rebuilding storage from disk.");
+    debugs(20, Important(47), "  " << std::setw(7) << counts.scancount  << " Entries scanned");
+    debugs(20, Important(48), "  " << std::setw(7) << counts.invalid  << " Invalid entries.");
+    debugs(20, Important(49), "  " << std::setw(7) << counts.badflags  << " With invalid flags.");
+    debugs(20, Important(50), "  " << std::setw(7) << counts.objcount  << " Objects loaded.");
+    debugs(20, Important(51), "  " << std::setw(7) << counts.expcount  << " Objects expired.");
+    debugs(20, Important(52), "  " << std::setw(7) << counts.cancelcount  << " Objects cancelled.");
+    debugs(20, Important(53), "  " << std::setw(7) << counts.dupcount  << " Duplicate URLs purged.");
+    debugs(20, Important(54), "  " << std::setw(7) << counts.clashcount  << " Swapfile clashes avoided.");
+    debugs(20, Important(55), "  Took "<< std::setw(3)<< std::setprecision(2) << dt << " seconds ("<< std::setw(6) <<
```

Comment:
I think this should be a single cache.log message with many Debug::Extra lines. I bet that is what the original author would use if they had access to Debug::Extra. The number of use cases where admins will significantly benefit from being able to control individual lines here is close to zero, especially since the `Beginning Validation Procedure` message below is available if they want to detect the end of loading the index without getting all these details.

AFAICT, the only reason to enumerate all of these debugs() individually is the ability to claim that this PR does not change any default output at all. @yadij, do you think that reason is enough to prevent converting those additional debugs() lines into Debug::Extra details of the primary debugs() line in this PR?

----

N.B. Even if we do convert, the `setw(7)` calls should stay. This is not how the entries should be aligned, of course, but properly aligning all possible values is difficult, out of this PR scope and the complexity may not be desirable/wanted in cache.log code anyway.

Reply:
> The idea that this PR does not change the log output is bogus. The config parse exception handlers text is altered

Yes, sorry I forgot about those; I should have added "...except for bug fixes". To be honest, those fixes are a bit out of scope here (but I would not remove it unless others insist on that removal).

> So am for using Debug::Extra to make this a single block.

@eduard-bagdasaryan, it sounds like Amos and I agree that these debug lines should be merged into one using Debug::Extra. Please do that.

> Same for the debug output given IDs 43-45.

Sounds good to me.

Reply:
> Done for these and the Beginning Validation Procedure

Thank you. I am resolving this change request.

> (these paired messages should not be separated, I think).

They are paired indeed, but they carry a very different meaning and their timing is often very different. We should let the admin decide which independent messages they are interested so I restored a unique ID for the "Beginning Validation Procedure" step in be06f23.